regcore/tests/db_es_tests.py

```diff
@@ -184,3 +185,18 @@ def test_put(self):
                           'old_version': 'oldold',
                           'new_version': 'newnew',
                           'diff': {'some': 'structure'}})
+
+
+class ESPreamblesTest(TestCase, ESBase):
+    def test_get_404(self):
+        with self.expect_get('preamble', 'docdoc'):
+            self.assertIsNone(ESPreambles().get('docdoc'))
+
+    def test_get_success(self):
+        with self.expect_get('preamble', 'docdoc', {'arbitrary': True}):
+            self.assertEqual(ESPreambles().get('docdoc'), {'arbitrary': True})
+
+    def test_put(self):
+        with self.expect_put('preamble', 'docdoc'):
+            ESPreambles().put('docdoc', {"some": "structure"})
+        self.assertEqual(self.call_args[0][2], {"some": "structure"})
```

Comment:
I didn't notice this earlier, but I don't love storing `call_args` on `self`. I wonder if we could stick this on the context manager instead so that it can't leak across tests. Anyway, very minor.